modules/cluster-logging-uninstall.adoc

```diff
@@ -15,17 +15,96 @@ You can remove cluster logging from your cluster.
 
 To remove cluster logging:
 
-. Use the following command to remove everything generated during the deployment.
+. Use the following command to remove the Cluster Logging CR.
 +
 [source,terminal]
 ----
 $ oc delete clusterlogging instance -n openshift-logging
 ----
 
-. Use the following command to remove the Persistent Volume Claims that remain
-after the Operator instances are deleted:
+. Optional: Use the following command to remove any Cluster Logging Persistent Volume Claims:
 +
```

Comment:
For 4.6+,[optional] need to delete ClusterLogForwarder CR, if you had enabled ClusterLogForwarder.
oc delete ClusterLogForwarder instance -n openshift-logging